src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/MasterServiceMetricsTests.java

```diff
@@ -89,7 +86,7 @@ public void testGetMetricsPath() {
     public void testCollectMetrics() {
         masterServiceMetrics.collectMetrics(startTimeInMills);
         String jsonStr = readMetricsInJsonString(1);
-        assertTrue(jsonStr.contains(MasterPendingValue.Constants.PENDING_TASKS_COUNT_VALUE));
+        assertFalse(jsonStr.contains(MasterPendingValue.Constants.PENDING_TASKS_COUNT_VALUE));
```

Comment:
why this assert has changed?

Reply:
In this test case we have 0 pending tasks
Earlier tmp file looks like 
```
pending_tasks/current/metadata
{"current_time":1610945977204}
{"Master_PendingQueueSize":0}$
```
and currently we have added task type if total pending tasks are 0 then tmp file looks like
```
pending_tasks/current/metadata
{"current_time":1610945977204}$
```
This is the reason for assert change